addons/mail/models/mail_message.py

```diff
@@ -1007,11 +1007,22 @@ def _message_format(self, fnames):
                     })
 
             if message_sudo.model and message_sudo.res_id:
-                record_name = self.env[message_sudo.model] \
+                record = self.env[message_sudo.model] \
                     .browse(message_sudo.res_id) \
                     .sudo() \
-                    .with_prefetch(thread_ids_by_model_name[message_sudo.model]) \
-                    .display_name
+                    .with_prefetch(thread_ids_by_model_name[message_sudo.model])
+                record_name = record.display_name
+
+                if fetch_followers:
+                    followers = [{
+                        'email': follower.email,
+                        'id': follower.id,
+                        'is_active': follower.is_active,
+                        'is_editable': True,
+                        'name': follower.name,
+                        'partner_id': follower.partner_id.id,
```

Comment:
It would be nice to have a common format method for followers, common to `read_followers` route and this new usage.

Please also refer to that route for how to check access rights, if necessary in regard to my previous comment.